src/web/app/pages-instructor/instructor-track-view-page/instructor-track-view-page.component.ts

```diff
@@ -273,9 +275,9 @@ export class InstructorTrackViewPageComponent implements OnInit {
           let dataStyle: string = 'font-family:monospace; white-space:pre;';
           if (student.email in this.studentToLog) {
             const entry: FeedbackSessionLogEntry = this.studentToLog[student.email];
-            status = `Viewed last at   ${this.timezoneService.formatToString(entry.timestamp, log.feedbackSessionData.timeZone, 'ddd, DD MMM, YYYY hh:mm:ss A')}`;
+            status = `Viewed last at   ${this.timezoneService.formatToString(entry.timestamp, log.feedbackSessionData.timeZone, InstructorAuditLogsPageComponent.LOG_DATE_TIME_FORMAT)}`;
```

Comment:
I might prefer to separately define it here again, rather than import it from audit logs page! Makes sense to me to avoid coupling the two components